src/prune/scene/world_scene.hpp

```
@@ -61,7 +66,12 @@ namespace prune {
 
         void draw_debug_tools();
 
+        void begin_runtime();
+
         virtual void update_runtime(float dt, const Input& input, bool keyboard_input_enabled) = 0;
```

Clang (14.0.6)

[warning] parameter name 'dt' is too short, expected at least 3 characters

(readability-identifier-length)